spec/acceptance/windows_spec.rb

```diff
@@ -2,8 +2,13 @@
 require 'spec_helper_acceptance'
 
 describe 'windows exec task', if: fact_on(default, 'osfamily') == 'windows' do
```

Comment:
Probably worth using os[:family] here.

Reply:
Going to leave these to get done as part of the acceptance audit